src/PrettyOrdering.jl

```diff
@@ -0,0 +1,46 @@
+pretty_lt(x, y) = false
```

Comment:
I think this is a really bad default, as it relies a lot on julia's sort how to handle non-orderings.
What about 
```suggestion
pretty_lt(x, y) = string(x) < string(y)
```
instead?